frontend/src/components/Collections/components/CollectionsGrid/components/CollectionRow/components/Popover/index.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Button,
+  Intent,
+  Popover as PopoverRaw,
+  Position,
+} from "@blueprintjs/core";
+import React, { FC } from "react";
+import { LeftAlignedDetailsCell } from "../common/style";
```

Comment:
To avoid circular dependency such as a parent importing a child, and the child imports something from the parent, I split out common styles in `/common/style`. And this way it's more explicit what styles are shared, so when we update code in common styles, we are naturally aware of its potential side effects